udata_gouvfr/settings.py

```diff
@@ -0,0 +1,11 @@
+"""
+Default settings for udata-gouvfr
+"""
+
+# Frontend banner parameters
+BANNER_ACTIVATED = False
```

Comment:
I added the requested PR on udata repo to erase those settings from udata itself : [#PR here](https://github.com/opendatateam/udata/pull/2507)